docs/source/references/api_spec.md

```diff
@@ -30,39 +33,47 @@
 **Method**: `POST`
 
 **Expects**:
+
 - Body: [`VersionedSignedQueryRequest`](#iroha-structures)
 - Query parameters:
-  + `start`: Optional parameter in queries where results can be indexed. Use to return results from specified point. Results are ordered where can be by id which uses rust's [PartialOrd](https://doc.rust-lang.org/std/cmp/trait.PartialOrd.html#derivable) and [Ord](https://doc.rust-lang.org/std/cmp/trait.Ord.html) traits.
-  + `limit`: Optional parameter in queries where results can be indexed. Use to return specific number of results.
-  + `sort_by_metadata_key`: Optional parameter in queries. Use to sort results containing metadata with a given key.
+    - `start`: Optional parameter in queries where results can be indexed. Use to return results from specified point.
+      Results are ordered where can be by id which uses
```

Comment:
something's not correct here?

Reply:
Nothing incorrect, that is the decision of Markdown formatted I used.

Reply:
If you want us to use prettier, we should probably check doc formatting in CI. Should we add a ticket for that?

Reply:
On that note, the wrap column should be set to 60, rather than 80 for markup.

Reply:
> If you want us to use prettier

No, I don't. I just used it once to fix ugly formatting. In general I don't think it is worth to setup a CI to check docs formatting, and manual formatting is just find (until the document eventually becomes ugly again)

Reply:
> WDYM? I understand the sentence, but am confused by "for markup".

Subtle point. Typographers work with 80 column-wide screens but the actual fill-column threshold for optimal viewing is 60 characters for text. We can get away with 80 in programming mostly because most well-indented programs are 90% blank space. 

> No, I don't. I just used it once to fix ugly formatting. In general I don't think it is worth to setup a CI to check docs formatting, and manual formatting is just fine (until the document eventually becomes ugly again)

OK.